src/providers/local/join-pool.provider.ts

```diff
@@ -53,7 +54,10 @@ export type AmountIn = {
  *
  * Handles pool joining state and transaction execution.
  */
-export const joinPoolProvider = (pool: Ref<Pool>) => {
+export const joinPoolProvider = (
+  pool: Ref<Pool>,
+  queryJoinDebounceMillis = 1000
```

Comment:
I don't understand, this change is already on master & develop, how is that possible? https://github.com/balancer/frontend-v2/blob/develop/src/providers/local/join-pool.provider.ts#L59